src/templates/next/layouts/SearchSG/SearchSG.tsx

```diff
@@ -0,0 +1,33 @@
+import { SearchSGPageSchema } from "~/engine"
+import { Skeleton } from "../Skeleton"
+
+const SearchSGLayout = ({
+  site,
+  page,
+  LinkComponent = "a",
+  HeadComponent = "head",
+  ScriptComponent = "script",
+}: SearchSGPageSchema) => {
+  const clientId =
+    (site.search && site.search.type === "searchSG" && site.search.clientId) ||
+    ""
+
+  return (
+    <Skeleton
+      site={site}
+      page={page}
+      LinkComponent={LinkComponent}
+      HeadComponent={HeadComponent}
+      ScriptComponent={ScriptComponent}
```

Comment:
can help me to add these to the other layouts 😄

Reply:
Yeah, added in 8ddd83a134333956693f634b012c8370f70adeca.